### PR TITLE
Adding more information to readme

### PR DESCRIPTION
**Pull request by munyirik**

### CODE REVIEW

**Comment by munyirik** [comment]
@alanch-ms please CR. Just minor updates to the readme.

File: README.md [hunk 1]
```diff
@@ -2,10 +2,34 @@
...
+    res.end(String(date));
+}).listen(1337);
+
+uwp.close();
+```
 
 ##Try it out!
 Follow the instructions [here](http://ms-iot.github.io/content/en-US/win10/samples/NodejsWU.htm) for the steps to install the required software and to build your first 'Hello World' Node.js server.
-You can also try out the [Blinky sample](http://ms-iot.github.io/content/en-US/win10/samples/NodejsWUBlinky.htm) which shows the use of WinRT APIs inside a Node.js server to toggle an LED on or off.
+You can also try out the [Blinky sample](http://ms-iot.github.io/content/en-US/win10/samples/NodejsWUBlinky.htm) which shows the use of UWP APIs inside a Node.js server to toggle an LED on or off.
```
Comment (alanch-ms) [inline README.md]: Minor: Any reason why "Blinky sample" is special?  Should it be in the "Other samples to try out:" section?

Comment (munyirik) [inline README.md]: It's a little special since I think it's the 'hello world' for IoT Core in other samples like C#.

**Comment by alanch-ms** [comment]
:+1:

**Comment by munyirik** [comment]
Thanks @alanch-ms

**Pull request merged**